articles/2016-09-01-semantic-html-tags.md

```diff
@@ -0,0 +1,88 @@
+# :name_badge: Semantic HTML Elements
+
+For this article, I assume you have a basic working knowledge of HTML. If this is not the case, please [email me](mailto:maddie.horowitz@addepar.com) and I will bring you up to speed.
```

Comment:
I agree that this could potentially get out of hand, but at the same time just sending them to a generic tutorial may not be as useful?